src/Goerli-DssSpell.sol

```diff
@@ -70,11 +71,13 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
 
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        // onboardCollaterals();
+        onboardCollaterals();
+        // updateCollaterals();
         // offboardCollaterals();
+        DssExecLib.setChangelogVersion("1.14.4");
     }
 }
 
 contract DssSpell is DssExec {
     constructor() DssExec(block.timestamp + 30 days, address(new DssSpellAction())) public {}
-}
+}
```

Comment:
Don't trim newlines at file end. There's probably an editor setting you can adjust for this.

Reply:
I'd support the addition of an `.editorconfig` file for sure.